presto-hudi/pom.xml

```diff
@@ -222,5 +222,24 @@
             <artifactId>units</artifactId>
             <scope>provided</scope>
         </dependency>
+
+        <!--should remove lz4 and caffeine, once bump hudi to 0.13-->
+        <dependency>
```

Comment:
medataTable need lz4,  schema evolution need caffeine.

Reply:
let's create a tracking ticket in Hudi JIRA.